scripts/automated-translations.ps1

```diff
@@ -146,9 +158,13 @@ else
     else
     {
       Write-Output "`n➡︎ Creating a pull request for changes"
-      Write-Output "`n# gh pr create --base $TranslationBranchName --head $LtsBranchName --title '${CommitMessage}'"
-      gh pr create --base $TranslationBranchName --head $LtsBranchName --title "${CommitMessage}"
-      $prForChanges = gh pr list --json title,url,headRefName --jq ".[] | select(.headRefName == `"${LtsBranchName}`")"
+      Write-Output "`n# gh pr create --base $LtsBranchName --head $TranslationBranchName --title '${CommitMessage}'"
+      gh pr create --base $LtsBranchName --head $TranslationBranchName `
+        --title "${CommitMessage}" `
+        --body ":robot: This pull request was created by the automated translations script." `
+        --label "risk level (author): 1" `
+        --label "skip e2e"
+      $prForChanges = gh pr list --json title,url,headRefName --jq ".[] | select(.headRefName == `"${TranslationBranchName}`")"
```

Comment:
This was another major issue: the head and base parameters were flipped, and the body parameter was missing.